epidose/device/util.sh

```diff
@@ -129,6 +136,54 @@ run_python()
   "$PYTHON" "$SCRIPT_DIR"/"$prog".py $DEBUG_FLAG $VERBOSE_FLAG "$@"
 }
 
+# Check the validity of the cuckoo filter
+# preconditions: none 
+# Returns, through the stdout,
+# the time until a Cuckoo filter is valid (in seconds)
+# If 0 is returned, the Cuckoo filter is stale.
```

Comment:
An output value of 0 means that the Cuckoo filter is stale, right?

Reply:
Yes.